IanaRegistrations/sarif-media-type.txt

```diff
@@ -6,16 +6,24 @@ Required parameters: N/A
 
 Optional parameters: N/A
 
-Encoding considerations: UTF8 only
+Encoding considerations: Binary: UTF8-encoded text only
 	
 Security considerations:
```

Comment:
>Security [](start = 0, length = 8)

In addition to the changes I incorporated, you made one comment on this section that I didn't understand. Could you please clarify:

> Run information can also be present, and it can include details about when analysis runs are performed. An attacker might be able to leverage this to avoid getting caught doing something naughty.

Reply:
The reviewer replied:

> My point was that timing information tells you when scans are performed, and knowing that might help an attacker making unauthorized change avoid scans that would detect those changes.

I see. So for example, if you know that scans are performed every Friday, then you can make an evil change on Monday, revert it on Thursday, get the benefit of four days of evil, and not get caught by the next scan.

---
In reply to: [453729336](https://github.com/oasis-tcs/sarif-spec/pull/465#discussion_r453729336) [](ancestors = 453729336)